packages/next/client/dev/send-client-error-to-dev-server.ts

```diff
@@ -0,0 +1,13 @@
+export default function sendClientErrorToDevServer(
```

Comment:
I think we can send this over the `websocket` instead, an example of this being done can be seen here https://github.com/vercel/next.js/blob/2608e7d86504dd4be41424dcb6f75493b7f6f9cf/packages/next/client/dev/error-overlay/hot-dev-client.js#L196